_custom_build/backend.py

```diff
@@ -0,0 +1,31 @@
+import sys
+
+from setuptools.build_meta import *  # noqa: F401, F403
+from setuptools.build_meta import _BuildMetaBackend
```

Comment:
Hmm, I'm not too keen on using an internal class from setuptools, especially as there have been quite big changes in the stable releases, and `_BuildMetaBackend` could change without notice.

Reply:
Rather than importing `_BuildMetaBackend`, would
```python
backend_class = build_wheel.__self__.__class__


class _CustomBuildMetaBackend(backend_class):
```
be better?

Reply:
I guess it should be more stable, but really I'd like upstream setuptools to be fixed :)  But that's seems unlikely to be ready before 1st July, so I pinged the issue (https://github.com/pypa/setuptools/issues/2491#issuecomment-1589764230) to see if anyone there has a nicer solution. If not, let's go with this!

Reply:
People over at https://github.com/pypa/setuptools/issues/2491 like this, so let's go for it, thanks!